docassemble/AssemblyLine/data/questions/al_terms_of_use.yml

```diff
@@ -0,0 +1,72 @@
+---
+code: |
+  # Customize this with the preferred contact information. It could be a physical address
+  # rather than an email.
+  AL_DEFAULT_CONTACT_INFORMATION = get_config("server administrator email", AL_ORGANIZATION_TITLE)
```

Comment:
nit: IMO `AL_ORG_CONTACT_INFO` is a better name, it's not clear what it's the default for